### DISCUSSION TITLE
MaterialFX Compatibility with latest JavaFX Version

### DISCUSSION THREAD

**Discussion by lokiuucx** [palexdev/MaterialFX#385] [General]
First of all, big fan of this library!

I have recently update my project's dependencies to:
javafx version: 23-ea+22
jdk22

After this updates, the MaterialFX Controls Design is not working properly (seems to render the MaterialFX Control with the standard JavaFX Control Design inside):

![Bildschirmfoto vom 2024-08-02 09-48-33](https://github.com/user-attachments/assets/fdbcc41c-935f-4a27-9ccf-d2ec735d534f)

Is the latest MaterialFX (version 11.16) not compatible with the latest javafx version?

**Comment by JoanCollaguazo** [comment]
Tengo el mismo problema, ¿existe alguna solución?

**Comment by jeftedariel** [comment]
Hi, im having same issue here, it got displayed well in scene builder, but it doesnt while im running the program

**Comment by FelipeAumannRS** [comment]
@jeftedariel @JoanCollaguazo @lokiuucx 

Hey, hope you guys still have an interest on fixing it :)

I'm using java and javafx version 24, materialsfx 11.17.0

I have faced this kind of issue before, all you have to do is setting materialsFX node css directly to it, also setting its style class (e.g .mfx-combo-box)

If you need more guidance on it, post a comment marking me with the node class (MFXButton, MFXTextField, MFXCircleToggleNode, etc) and i'll be posting a fix

**Reply by jishimwe** [reply]
Hello,
I am having the same issue and I can't seem to find a fix.

Everything looks great with MaterialFX in the scene builder. But when I run it, it reverts to default JavaFX style.
